sysutils/monit/src/opnsense/mvc/app/models/OPNsense/Monit/Monit.xml

```diff
@@ -46,6 +46,57 @@
             <default>0</default>
             <Required>Y</Required>
          </ssl>
+         <logfile type="TextField">
+            <Required>N</Required>
+            <default>syslog facility log_daemon</default>
+            <mask>/^(\/[^\/ ]*)+\/?|syslog.*$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the log file or the keyword "syslog".</ValidationMessage>
+         </logfile>
+         <statefile type="TextField">
+            <Required>N</Required>
+            <mask>/^(\/[^\/ ]*)+\/?$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the state file.</ValidationMessage>
+         </statefile>
+         <eventqueuePath type="TextField">
+            <Required>N</Required>
+            <mask>/^(\/[^\/ ]*)+\/?$/</mask>
+            <ValidationMessage>Should be a valid absolute path to the eventqueue directory.</ValidationMessage>
+         </eventqueuePath>
+         <eventqueueSlots type="IntegerField">
+            <Required>N</Required>
+            <ValidationMessage>Eventqueue Slots must be a numeric value.</ValidationMessage>
+         </eventqueueSlots>
+         <httpdEnabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
```

Comment:
if you require this, please bump the model version to 1.0.1 above

Reply:
What exactly do you mean?

Reply:
this means you require this field to be set to "0" by default, but if you don't start a model migration it won't be there after install.. look for &lt;version&gt; and increase it :)

Reply:
Ah, cool! Got it.